.changeset/thirty-bobcats-change.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/fields': major
```

Comment:
Even though the keystone package wasn't changed, this is transitively a major change

```suggestion
'@keystone-next/fields': major
'@keystone-next/keystone': major
```